src/main/io/displayport_msp.c

```diff
@@ -109,7 +109,12 @@ static int writeString(displayPort_t *displayPort, uint8_t col, uint8_t row, uin
     buf[0] = 3;
     buf[1] = row;
     buf[2] = col;
-    buf[3] = displayPortProfileMsp()->attrValues[attr];
+    buf[3] = displayPortProfileMsp()->attrValues[attr & ~DISPLAYPORT_ATTR_BLINK] & DISPLAYPORT_MSP_ATTR_MASK;
```

Comment:
`buf[3] = displayPortProfileMsp()->attrValues[attr] & ~DISPLAYPORT_MSP_ATTR_BLINK & DISPLAYPORT_MSP_ATTR_MASK` looks a bit cleaner to me as it does all the masking on the same side of the lookup.

Reply:
The mask for the index `attr & ~DISPLAYPORT_ATTR_BLINK` is to shave off possible blink bit in the config value, and the `DISPLAYPORT_MSP_ATTR_MASK` mask is to drop undefined bits in (normal) attributes, so your suggestion is semantically different but results are the same. I'm wondering ...

Reply:
Yep, it is the same. When trying to figure out what it does I find bit masking only on one side of the array lookup is easier.

Reply:
👍